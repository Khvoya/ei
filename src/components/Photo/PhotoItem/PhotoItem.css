.PhotoItem {
  cursor: pointer;
  max-height: 200px;
  min-width: 300px;
}
.PhotoItem:focus,
.PhotoItem:active,
.Photo__wrap button:focus,
.Photo__wrap button:active {
  outline: none;
}
