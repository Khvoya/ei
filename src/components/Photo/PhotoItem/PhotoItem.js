import React from "react";
import "./PhotoItem.css";
import PropTypes from "prop-types";

const PhotoItem = props => {
  return <img alt="photoItem" className="PhotoItem" src={props.photo} />;
};

PhotoItem.propTypes = {
  photo: PropTypes.string.isRequired
};

export default PhotoItem;
