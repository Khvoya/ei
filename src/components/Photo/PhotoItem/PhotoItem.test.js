import React from 'react';
import { shallow } from 'enzyme';
import { shallowToJson } from 'enzyme-to-json';
import PhotoItem from './PhotoItem';

describe('PhotoItem', () => {

  it('should render correctly', () => {
    const output = shallow(
      <PhotoItem />
    );
    expect(shallowToJson(output)).toMatchSnapshot();
  });
});
