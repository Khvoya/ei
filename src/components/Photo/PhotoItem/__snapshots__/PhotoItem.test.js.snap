// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PhotoItem should render correctly 1`] = `
<img
  alt="photoItem"
  className="PhotoItem"
/>
`;
